src/msg/async/Event.h

```diff
@@ -113,7 +113,7 @@ class EventCenter {
   CephContext *cct;
   int nevent;
   // Used only to external event
-  pthread_t owner;
+  pthread_t owner = 0;
```

Comment:
so when `in_thread()` gets called, `owner` is always initialized. and how come we have this valgrind warning?